docs/en/stack/ml/nlp/ml-nlp-ner-example.asciidoc

```diff
@@ -0,0 +1,296 @@
+This example uses an 
+https://huggingface.co/elastic/distilbert-base-uncased-finetuned-conll03-english[uncased NER model].
+
+Install the model by running the `eland_import_model_hub` command in the Docker 
+image:
+
+[source,shell]
+--------------------------------------------------
+docker run -it --rm elastic/eland \
+    eland_import_hub_model \
+      --cloud-id $CLOUD_ID \
+      -u <username> -p <password> \
+      --hub-model-id elastic/distilbert-base-uncased-finetuned-conll03-english \
+      --task-type ner \
+      --start
+
+--------------------------------------------------
+
+You need to provide an administrator username and its password and replace the 
+`$CLOUD_ID` with the ID of your Cloud deployment. This Cloud ID can be copied 
+from the deployments page on your Cloud website.
+
+include::ml-nlp-shared.asciidoc[tag=nlp-start]
+
+include::ml-nlp-shared.asciidoc[tag=nlp-sync]
+
+
+[discrete]
+[[ex-ner-test]]
+== Testing the NER model
+
+Deployed models can be evaluated using the 
+{ref}/infer-trained-model-deployment.html[_infer API]. The input is the string 
+you want to analyze. In the following request, `text_field` is the field name 
+where the model expects to find the input, as defined in the model 
+configuration. By default, if the model was uploaded via Eland, the input field 
+is `text_field`.
+
+[source,js]
+--------------------------------------------------
+POST _ml/trained_models/elastic__distilbert-base-uncased-finetuned-conll03-english/_infer
+{
+  "docs": [
+    {
+      "text_field": "Elastic is headquartered in Mountain View, California."
+    }
+  ]
+}
+--------------------------------------------------
+
+The model finds three entities: an organization "Elastic", and two locations 
+"Mountain View" and "California" as the response shows:
+
+[source,js]
+--------------------------------------------------
+{
+  "inference_results": [
+    {
+      "predicted_value": "[Elastic](ORG&Elastic) is headquartered in [Mountain View](LOC&Mountain+View), [California](LOC&California).",
+      "entities": [
+        {
+          "entity": "elastic",
+          "class_name": "ORG",
+          "class_probability": 0.9958921231805256,
+          "start_pos": 0,
+          "end_pos": 7
+        },
+        {
+          "entity": "mountain view",
+          "class_name": "LOC",
+          "class_probability": 0.9844731508992688,
+          "start_pos": 28,
+          "end_pos": 41
+        },
+        {
+          "entity": "california",
+          "class_name": "LOC",
+          "class_probability": 0.9972361009811214,
+          "start_pos": 43,
+          "end_pos": 53
+        }
+      ]
+    }
+  ]
+}
+--------------------------------------------------
+// NOTCONSOLE
+
+
+[discrete]
+[[ex-ner-ingest]]
+== Adding the NER model to an {infer} ingest pipeline
+
+You can perform bulk {infer} on documents as they are ingested via an ingest 
```

Comment:
"via" is one of the terms discouraged in our inclusivity guidelines, so it could be reworded like this:

>... as they are ingested by using an {infer} processors in your ingest pipeline.